src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -126,25 +126,22 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
 
         if (!state_) {
             std::ostringstream buffer;
-            buffer << service_name << ": cannot call `infer` on a stopped or unconfigured triton service";
+            buffer << service_name
```

Comment:
There are some drive-by reformattings that happened when I ran clang-format folded in here.